source/Coop/Mod/Persistence/RPC/EventMethodCall.cs

```diff
@@ -51,7 +51,8 @@ protected override void Execute(RailRoom room, RailController sender)
                 if (room is RailServerRoom serverRoom)
                 {
                     Logger.Trace("[{eventId}] Broadcast SyncCall: {call}", EventId, Call);
-                    m_EnvironmentServer.EventQueue.Add(serverRoom, this);
+                    if(!m_EnvironmentServer.EventQueue.Add(serverRoom, this))
```

Comment:
Logic error. Shouldn't be inverted, should it?

Reply:
Yes my bad, I've fixed it.

Reply:
I can't see the commit, did you push your changes?

Reply:
No I'm going a complete different route to be sure the game will pause before the overflow, but it's not yet functionnal so i haven't pushed it. In fact I have a hard time testing if my solution actually works, I've tried to use Logger.Debug() to write some info on screen (for example the size of the queue), but I doesn't seem to print the result anywhere I've looked.

Reply:
Hi @mihien 

You can view the queue size in the debug info (Ctrl + `):
![image](https://user-images.githubusercontent.com/71883114/100143583-70a41500-2e8d-11eb-97e4-7196a6debe13.png)

Thanks
Jack

Reply:
Ah so that's where it was hidden, it's going to be insanely easier to work with this haha 
Thank you!